test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-sale_financial_risk @ git+https://github.com/OCA/credit-control.git@16.0#subdirectory=setup/sale_financial_risk
```

Comment:
Could you drop this file in order to have the CI tests pass? 🙏🏻 

Normally this isn't needed anymore, as dependencies have been merged